addons/l10n_ec/models/account_journal.py

```diff
@@ -0,0 +1,18 @@
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+
+    _inherit = "account.journal"
+
+    l10n_ec_entity = fields.Char(string="Emission Entity", size=3, default="001")
+    l10n_ec_emission = fields.Char(string="Emission Point", size=3, default="001")
+    l10n_ec_company_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        related="company_id.partner_id",
+    )
+    l10n_ec_emission_address_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Emission address",
+        domain="['|', ('id', '=', l10n_ec_company_partner_id), '&', ('id', 'child_of', l10n_ec_company_partner_id), ('type', '!=', 'contact')]",
```

Comment:
This makes that the partner is part of the company (It can be restrictive, so good to check if this works in all of the cases)

Reply:
for VBE FUN test to check (use commercial partner and not partner) (domain on the field) (with diff VAT numbers you will need diff companies

Reply:
We keep domain to ensure you have an address to belong main company